spec/rails_helper.rb

```diff
@@ -73,7 +73,7 @@
 RSpec.configure do |config|
   config.include RSpecHtmlMatchers
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = "#{::Rails.root}/test/fixtures"
+  config.file_fixture_path = "#{::Rails.root}/test/fixtures"
```

Comment:
I can't remember, why this was needed... :thinking: 
But for sure it was :grimacing: